wfe2/wfe.go

```diff
@@ -938,7 +938,7 @@ func (wfe *WebFrontEndImpl) parseRevocation(
 
 	// Try to validate the signature on the provided cert using its corresponding
 	// issuer certificate.
-	issuerNameID := issuance.GetIssuerNameID(parsedCertificate)
+	issuerNameID := issuance.IssuerNameID(parsedCertificate)
 	issuerCert, ok := wfe.issuerCertificates[issuerNameID]
```

Comment:
Nit: You could change line 942 to the following and remove line 941. The `issuerNameID` variable is only used in this one instance.
```
issuerCert, ok := wfe.issuerCertificates[issuance.IssuerNameID(parsedCertificate)]
```